challenge.js

```diff
@@ -34,14 +34,14 @@
  *	Using the same HTML elements as above...
  *
  *	Suppose A is the body element, B, C, and D are all divs,
- *		and E is a p. 
+ *		and E is a p.
  *
  *	Write a CSS selector that can be used to retrieve E from the DOM.
  *
  *	Answer below.
  */
 
-// ANSWER
+// ANSWER  document.getElementById."E"
```

Comment:
You were not given element IDs to select, but you were given tag names. As a side note, what you wrote is not a function invocation, but a syntactically incorrect property access. Here is an explanation and sample solution:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.